cfme/dashboard.py

```diff
@@ -385,7 +385,7 @@ def all(self, content_type=None):  # widgets
         view = self.dashboard_view
         result = []
         # TODO: Idiomatize the following line
-        for (widget_name, ) in view.widgets.view_class.all(view.browser):
+        for (widget_name, ) in view.widgets.all(view.browser):
```

Comment:
Surprised this does work without passing a TITLE parameter to `view.widgets`, since its still decorated as a nested 
parametrized view